tests/functional/workloads/cnv/test_vm_snapshot_cloning_ops.py

```diff
@@ -64,9 +64,42 @@ def test_vm_clone(self, cnv_workload, clone_vm_workload, setup_cnv):
             clone_obj.stop()
 
     @workloads
-    @pytest.mark.polarion_id("OCS-6299")
+    @pytest.mark.parametrize(
+        argnames=["pvc_expand_before_snapshot", "pvc_expand_after_restore"],
+        argvalues=[
+            pytest.param(
+                False,
```

Comment:
Do we really need this? IIUC, the code still covers snapshot create and restore without this. Please check if we can remove this.